ibis/expr/types/relations.py

```diff
@@ -3611,6 +3611,189 @@ def pivot_wider(
 
         return self.group_by(id_cols).aggregate(**aggs)
 
+    def relocate(
+        self,
+        *selectors: str | s.Selector,
+        before: str | s.Selector | None = None,
+        after: str | s.Selector | None = None,
+        **kwargs: str | s.Selector,
+    ) -> Table:
+        """Relocate columns in a table.
+
+        Parameters
+        ----------
+        selectors
+            Selectors to relocate.
+        before
+            A column name or selector to insert the new columns before.
+        after
+            A column name or selector to insert the new columns after.
```

Comment:
Can you add a clarification here on what before/after a selector means? It's clear if the selector returns one column, but unclear what it means if it returns more than one. Perhaps something like

```suggestion
            A column name or selector. Columns in `columns` are relocated after the last
            column selected in `after`.
```